### PR TITLE
Changeling megabuffs.

### PR DESCRIPTION
**Pull request by chaoko99**
🆑 
Added the armblade from Urist.
Added the changeling shield from Aurora.
Added Epi chem to the Epi sacks ability.
Added D+ to the Epi sacks ability, to get you out of Oxycrit.
added the NO_EMBED obj flag. Added it to the armblade and the Esword.
🆑

### CODE REVIEW

**Comment by chaoko99** [comment]
Thanks for the code, Urist.

File: code/game/gamemodes/changeling/changeling_powers.dm [hunk 1]
```diff
@@ -487,7 +487,7 @@ var/global/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","E
 /mob/proc/changeling_unstun()
 	set category = "Changeling"
 	set name = "Epinephrine Sacs (45)"
-	set desc = "Removes all stuns"
+	set desc = "Removes all stuns. Don't use it in rapid succession."
```
Comment (scrdest) [inline code/game/gamemodes/changeling/changeling_powers.dm:490]: Careful, descs can get truncated in the UI.

Comment (chaoko99) [reply code/game/gamemodes/changeling/changeling_powers.dm:490]: Checked, it's not too long.

**Comment by chaoko99** [comment]
This one's done, and testing seems to work.

**Pull request merged**